src/dotnet/commands/dotnet-help/HelpCommandParser.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Microsoft.DotNet.Cli.CommandLine;
+
+namespace Microsoft.DotNet.Tools.Help
+{
+    internal static class HelpCommandParser
+    {
+        public static Command Help()
+        {
+            return Create.Command(
+                "help",
+                LocalizableStrings.AppFullName,
+                Accept.ZeroOrOneArgument()
+                    .With(
+                        LocalizableStrings.CommandArgumentDescription,
+                        LocalizableStrings.CommandArgumentName),
+                Create.Option(
+                    "-h|--help|-?|/?",
```

Comment:
Sorry, this is not the existing behavior. 
Here is the existing behavior
_dotnet -h|--help|-?|/?_ is the same as _dotnet help_
and
(I) _dotnet help --help_ and _dotnet help -h_ will throw
(II) _dotnet help -?_ and _dotnet help /?_ will print
```
.NET CLI help utility

Usage: dotnet help [arguments]

Arguments:
  <COMMAND_NAME>  CLI command for which to view more detailed help.
```

I think the only thing need to change is to make (I) and (II) the same. 

Another problem, apparently /? | -? | --help | -h is not consistent across the board today, but most of the command have /?. 

Tried on today's master
has -?|/? and it is the same as -h|--help
dotnet restore 
dotnet migrate
dotnet run
dotnet test
dotnet build

does not have: -?|/? but has -h|--help
dotnet nuget